fabtests/benchmarks/benchmark_shared.h

```diff
@@ -40,7 +40,7 @@ extern "C" {
 
 #include <rdma/fi_rma.h>
 
-#define BENCHMARK_OPTS "vkj:W:"
+#define BENCHMARK_OPTS "vkKj:W:"
```

Comment:
The fork was added into common/shared.c, which should work for basically all tests.  Why restrict the option to only the benchmark tests then, versus adding this as a common option?

Reply:
I will add the `-K` option to `INFO_OPTS` in the newer revision.